src/core/config.py

```diff
@@ -7,6 +7,7 @@ class Config:
     GEMINI_MODEL = os.environ.get('GEMINI_MODEL', 'gemini-1.5-flash-002')
     OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')
     OPENAI_MODEL = os.environ.get('OPENAI_MODEL', 'gpt-4o-mini')
+    HUMAN_LANGUAGE = os.environ.get('HUMAN_LANGUAGE', 'en')
```

Comment:
This line adds a new environment variable `HUMAN_LANGUAGE` with a default value of `'en'`.  While this is functional, consider adding validation or a more robust way of handling this.  For example, what happens if an invalid language code is provided?  A list of supported languages, or a mechanism to check for valid language codes, would make the system more resilient.  Additionally, consider adding documentation explaining how to set this variable and what valid values are.

Also, consider the security implications of relying solely on environment variables.  If this variable is used to control access or filter sensitive data, this should be handled more securely.